tests/models/test_transformers.py

```diff
@@ -76,6 +76,23 @@ def test_transformer_smoke_select(model_name, model_kwargs):
     assert lm["answer"] in ["p", "t", "w"]
 
 
+@pytest.mark.needs_credentials
+@pytest.mark.skip("Need to figure out auth")
+def test_llama3_chat():
+    lm = models.Transformers(
+        r"meta-llama/Meta-Llama-3-8B-Instruct", trust_remote_code=True
+    )
+    with system():
+        lm += "You are a counting bot. Just keep counting numbers."
+    with user():
+        lm += "1,2,3,4"
+    with assistant():
+        lm += gen(name="five", max_tokens=10)
+
+    assert "5" in lm["five"]
```

Comment:
What is it about this test which is specific to Llama3? Shouldn't all models be able to pass?

Reply:
Nothing in particular, I was just using it to debug llama3. removed the test

Reply:
technically all the phi-3 tests should run on any model too, phi is just what they were designed for (as likely to result in errors due to tokenizer choices). the llama3 test was identical to phi tests so we should just generalize those in future PRs